docker/safety-rules/key-manager.sh

```diff
@@ -12,26 +12,26 @@ fi
 if [ -n "${CFG_CHAIN_ID}" ]; then
         params+="--chain-id ${CFG_CHAIN_ID} "
 fi
-if [ -n "${JSON_RPC_ENDPOINT}" ]; then
-    params+="--json-rpc-endpoint ${JSON_RPC_ENDPOINT} "
+if [ -n "${CFG_JSON_RPC_ENDPOINT}" ]; then
```

Comment:
are these defined anywhere else? do we need to update some k8s or terraform?

Reply:
'cc @mgorven. Any impact to these files? I can find one reference but not sure if it's breaking.

Reply:
k8s doesn't use these entrypoint files. The keymanager config file will need to be updated: https://github.com/libra/partners/blob/master/validator/helm/files/configs/keymanager.yaml

Reply:
Awesome, thanks! I'll update that repo too